tools/postgres/dbtool.js

```diff
@@ -0,0 +1,447 @@
+        .map((filename) => path.join(evolutionsPath, filename))
+        .sort(),
+    );
+    schemaDumpDir = dumpSchemaViaSqlFiles([schemaPath]);
+
+    cleanSchemaDump(evolutionsDumpDir);
+    cleanSchemaDump(schemaDumpDir);
+
+    try {
+      safeSpawn("diff", ["--strip-trailing-cr", "-r", evolutionsDumpDir, schemaDumpDir]);
+    } catch (err) {
+      throw new Error(`Evolutions do not match with schema file:\n${err.stdout}`);
+    }
+  } finally {
+    if (fs.existsSync(evolutionsDumpDir)) {
+      fs.rmSync(evolutionsDumpDir, { recursive: true, force: true });
+    }
+    if (fs.existsSync(schemaDumpDir)) {
+      fs.rmSync(schemaDumpDir, { recursive: true, force: true });
+    }
+  }
+  console.log("✨✨ Evolutions match with schema file");
+}
+
+function applyEvolutions() {
+  const schemaVersion = parseInt(
+    callPsql("SELECT schemaVersion FROM webknossos.releaseInformation;").trim(),
+    10,
+  );
+  if (isNaN(schemaVersion)) {
+    console.error("Error: Schema version is not a number");
+    process.exit(1);
+  }
+  console.log(`Schema version: ${schemaVersion}`);
+
+  // get list of evolutions to apply
+  const evolutions = fs
+    .readdirSync(evolutionsPath)
+    .filter((filename) => filename.endsWith(".sql"))
+    .map((filename) => {
+      const num = parseInt(filename.split("-")[0], 10);
+      return [num, filename];
+    })
+    .filter(([num]) => num > schemaVersion)
+    .sort((a, b) => a[0] - b[0]);
+
+  // apply evolutions
+  if (evolutions.length > 0) {
+    for (const [_, evolutionFilename] of evolutions) {
+      console.log(`Applying evolution: ${evolutionFilename}`);
+      safePsqlSpawn([
+        PG_CONFIG.url,
+        "-v",
+        "ON_ERROR_STOP=ON",
+        "-q",
+        "-f",
+        path.join(evolutionsPath, evolutionFilename),
+      ]);
+    }
+    console.log("✨✨ Successfully applied the evolutions");
+  } else {
+    console.log("There are no evolutions that can be applied.");
+  }
+}
+
+function assertUniqueEvolutionNumbers() {
+  const groupedEvolutions = new Map();
+  for (const filename of fs.readdirSync(evolutionsPath)) {
+    const num = parseInt(filename.split("-")[0], 10);
+    if (groupedEvolutions.has(num)) {
+      groupedEvolutions.get(num).push(filename);
+    } else {
+      groupedEvolutions.set(num, [filename]);
+    }
+  }
+
+  if (Array.from(groupedEvolutions.values()).some((group) => group.length > 1)) {
+    console.log("Duplicate evolutions found:");
+    for (const [num, filenames] of groupedEvolutions.entries()) {
+      if (filenames.length > 1) {
+        console.log(num, filenames);
+      }
+    }
+
+    process.exit(1);
+  }
+  console.log("✨✨ All evolution numbers are unique");
+}
+
+program.name("dbtool").description("Tool for managing the WEBKNOSSOS database");
+
+program
+  .command("drop-db")
+  .description("Drop the current database")
+  .action(() => {
+    dropDb();
+    console.log("✨✨ Done");
+  });
+
+program
+  .command("refresh-db")
```

Comment:
Maybe call it `drop-and-refresh-db`? I always found the existing naming a bit dangerous.